### ISSUE TITLE
No repo file is created

### ISSUE THREAD

**Issue by mortzel** [NeCTAR-RC/puppet-mariadb#18]
Getting following error when trieing to install a galera cluster
`Error: Failed to apply catalog: Could not find dependent Exec[apt_update] for File[/etc/apt/sources.list.d/mariadb.list] at /etc/puppet/modules/apt/manifests/setting.pp:63`

Puppet Version: 3.4.3-1
puppetlabs-apt: 2.1.0
Distro: Ubuntu 14.04/12.04

**Comment by sorrison** [comment]
Yeah I think this is due to the puppetlabs-apt module changing things around a bit. We are using it with puppetlabs-apt 1.8.0. Haven't looked into the newer apt version and what changes it needs. Happy to take a pull request to make it support it.

**Comment by mortzel** [comment]
With puppetlabs-apt 1.8.0 and `include stdlib` getting following error 
`(Anchor[apt::source::mariadb] => Apt::Source[mariadb] => Class[Mariadb::Repo::Debian] => Stage[setup] => Stage[main] => Class[Apt::Update] => Exec[apt_update] => Class[Apt::Update] => Anchor[apt::source::mariadb])`
